Source/WebKit/UIProcess/Extensions/WebExtensionContext.messages.in

```diff
@@ -48,6 +48,13 @@ messages -> WebExtensionContext {
     // Commands
     CommandsGetAll() -> (Vector<WebKit::WebExtensionCommandParameters> commands);
 
+    // Cookies
+    CookiesGet(std::optional<PAL::SessionID> sessionID, String name, URL url) -> (std::optional<WebKit::WebExtensionCookieParameters> cookie, std::optional<String> error);
```

Comment:
`WebKit::WebExtensionContext:ErrorString` instead of  ` std::optional<String> error`? more wordy but it might be better to use here

Reply:
I'll try.